src/components/FieldGroup/FieldGroup.js

```diff
@@ -22,15 +23,17 @@ const FieldGroup = props => {
   return (
     <div {...restProps} className={mergedClassNames}>
       {children}
+      {error && <FieldError>{error}</FieldError>}
```

Comment:
take a look at TextField component - maybe it's worth to add also the description :)

Reply:
added ;)